flang/include/flang/Optimizer/Builder/Factory.h

```diff
@@ -147,8 +147,10 @@ void genCharacterCopy(mlir::Value src, mlir::Value srcLen, mlir::Value dst,
 inline std::vector<mlir::Value> getExtents(mlir::Value shapeVal) {
   if (shapeVal)
     if (auto *shapeOp = shapeVal.getDefiningOp()) {
-      if (auto shOp = mlir::dyn_cast<fir::ShapeOp>(shapeOp))
-        return shOp.getExtents();
+      if (auto shOp = mlir::dyn_cast<fir::ShapeOp>(shapeOp)) {
+        auto operands = shOp.getExtents();
+        return {operands.begin(), operands.end()};
```

Comment:
I assume this is because the conversion operator went away.

We should investigate getting rid of the use of `std::vector` and replacing it with `llvm::SmallVector` in these cases. The LLVM coding standard is explicit on that point.

Reply:
That's right. I have the replacement of `std::vector` on my todo list ... but not on the top :-)